openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ListFilter.java

```diff
@@ -227,6 +228,20 @@ private String getTestSuiteTypeCondition() {
     }
   }
 
+  private String getEmptyTestSuiteCondition() {
+    String includeEmptyTestSuites = getQueryParam("includeEmptyTestSuites");
+    if (includeEmptyTestSuites == null || Boolean.parseBoolean(includeEmptyTestSuites)) {
+      // if we want to include empty test suites, then we don't need to add a condition
+      return "";
+    }
+
+    if (Boolean.TRUE.equals(DatasourceConfig.getInstance().isMySQL())) {
```

Comment:
@TeddyCr why are we looking at test results to filter out. The goal a test suite has 3 test cases on day 1 , later point someone deleted all those tests cases , now we have empty test suite. Instead of listing it in the DQ portal, we will not show them.
Currently if I create a test suite with a test case and not run it or Yet to run. I.e if I schedule it for next day , it won't get listed in DQ page

Reply:
and we have different definition of what includeEmptyTestSuite means in case of TableList https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java#L1897